lib/attrsets.nix

```diff
@@ -1094,6 +1094,44 @@ rec {
     attrs:
     map (name: f name attrs.${name}) (attrNames attrs);
 
+  /**
+    Call a function for each attribute in the given set and return
+    the concatenated results.
+
+    # Inputs
+
+    `f`
+
+    : A function, given an attribute's name and value, returns a list of new values.
+
+    `attrs`
+
+    : Attribute set to map over.
+
+    # Type
+
+    ```
+    # FIXME
+    concatMapAttrsToList :: (String -> a -> b) -> AttrSet -> [b]
```

Comment:
I am very confused by the type syntax, I'll need help to get this right (this is currently a copy-paste of `mapAttrsToList`).